.github/workflows/prebuild.yml

```diff
@@ -31,6 +31,16 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: lts/*
+      - uses: sethvincent/sound-ci-helpers@windows-updates
```

Comment:
Inline the necessary steps instead so the only third-party actions we depend on are the from https://github.com/actions. If the workflow becomes unwieldy we can always move stuff to separate scripts.

Reply:
The windows script in particular is a bit involved. For now I vendored this into .github/actions/virtual-audio-device.